textile/src/main/java/io/textile/textile/Cafes.java

```diff
@@ -16,17 +16,17 @@ public void register(String host, String token) throws Exception {
 
     public CafeSession session(String peerId) throws Exception {
         byte[] bytes = node.cafeSession(peerId);
-        return CafeSession.parseFrom(bytes);
+        return bytes != null ? CafeSession.parseFrom(bytes) : null;
     }
 
     public CafeSessionList sessions() throws Exception {
         byte[] bytes = node.cafeSessions();
-        return CafeSessionList.parseFrom(bytes);
+        return CafeSessionList.parseFrom(bytes != null ? bytes : new byte[0]);
```

Comment:
In contrast, in the case of "list" types that are returned, we always want to return a valid object, potentially with an empty `items` array property. 

The error I was seeing yesterday was caused by calling `parseFrom` with a `null` argument, so this passes in an empty byte array if `bytes` is `null`.